examples/step-86/step-86.cc

```diff
@@ -203,7 +203,7 @@ namespace Step86
     // the same degrees of freedom, but with zero values even if the boundary
     // values for the solution are non-zero; we will keep this modified set of
     // constraints in `homogeneous_constraints`.
-    AffineConstraints<double> hanging_nodes_constraints;
+    AffineConstraints<double> hanging_node_constraints;
```

Comment:
I also changed the name here because I stumbled over it. We might not always be consistent with names in deal.II, but apart from step-86 we never use `hanging_nodes_constraints`, but more than a thousand times the term `hanging_node_constraints`.